pkgs/development/tools/the-way/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "the-way";
+  version = "v0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "out-of-cheese-error";
+    repo = pname;
+    rev = version;
+    sha256 = "0vnqrlpmrc9mz1snd94w7rcxd5s7jfc84pl8j2hd01cvzfqjqjf3";
+  };
+
+  cargoSha256 = "0l5yanrlx0km82w7rpa9kq6sq9dnb40im0hddabdrgjc2934m19v";
+  doCheck = false;
```

do you mind adding a comment as to why tests were disabled?
```suggestion
  doCheck = false;
```

---

Added it as the comment inside the derivation.